### ISSUE TITLE
Cannot use todonotes with cleanthesis

### ISSUE THREAD

**Issue by JackDeSomeTrades** [henrikmidtiby/todonotes#33]
Hi, I've been trying to use the todonotes package with cleanthesis. I have been unable to compile my document with todonotes. If I comment out the package, everything works perfectly. 

My preamble:

```
\documentclass[%
    paper=A4,               % paper size --> A4 is default in Germany
    twoside=true,           % onesite or twoside printing
    openright,              % doublepage cleaning ends up right side
    parskip=half,           % spacing value / method for paragraphs
    chapterprefix=true,     % prefix for chapter marks
    11pt,                   % font size
    headings=normal,        % size of headings
    bibliography=totoc,     % include bib in toc
    listof=totoc,           % include listof entries in toc
    titlepage=on,           % own page for each title page
    captions=tableabove,    % display table captions above the float env
    chapterprefix=false,    % do not display a prefix for chapters
    appendixprefix=false,    % but display a prefix for appendix chapter
    draft=false,            % value for draft version
]{scrreprt}%
\usepackage[english]{babel} % babel system, adjust the language of the content
\PassOptionsToPackage{% setup clean thesis style
    figuresep=colon,%
    hangfigurecaption=false,%
    hangsection=true,%
    hangsubsection=true,%
    sansserif=false,%
    configurelistings=true,%
    colorize=full,%
    colortheme=bluemagenta,%
    configurebiblatex=true,%
    bibsys=biber,%
    bibfile=bib-refs,%
    bibstyle=ieee,%
    bibsorting=nty,%
}{cleanthesis}
\usepackage{cleanthesis}

\usepackage{amsthm}
\usepackage{amsfonts}
\usepackage{amsmath}
\usepackage{mathrsfs}
\usepackage{mathspec}
\usepackage{scrhack}
\usepackage{todonotes}
```
Leads to errors like: 

```
Undefined control sequence.\section
Illegal parameter number in definition of \@themark.
```

**Comment by henrikmidtiby** [comment]
That looks strange.
I have tried to reproduce the example, but with no luck.
Can you provide a minimal working example that triggers the error?

**Comment by JackDeSomeTrades** [comment]
Just adding 

`\usepackage{todonotes}`

 in the preamble of the main tex file of the [devel branch of cleanthesis](https://github.com/derric/cleanthesis/tree/develop)  triggers this error.

**Comment by henrikmidtiby** [comment]
I finally managed to reproduce the issue. The clue was to use the development branch of cleanthesis.
After having reproduced the error, I have removed everything from the document, that did not relate to the issue and in that process, I created the following minimal working example of the problem.

```
\documentclass{scrreprt}

\usepackage{cleanthesis}
\usepackage{calc}

\begin{document}
\chapter{Chap}
\section{Sec 1}
\end{document}
```

Based on the minimal example, I conclude that the issue is not within the todonotes package, but either in the cleanthesis development branch or in the calc package.
